nusantara/nusa_datasets/id_hsd_nofaaulia/id_hsd_nofaaulia.py

```diff
@@ -0,0 +1,178 @@
+
+import datasets
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import (DEFAULT_NUSANTARA_VIEW_NAME,
+                                       DEFAULT_SOURCE_VIEW_NAME, Tasks)
+
+_CITATION = """\
+@inproceedings{10.1145/3330482.3330491,
+author = {Aulia, Nofa and Budi, Indra},
+title = {Hate Speech Detection on Indonesian Long Text Documents Using Machine Learning Approach},
+year = {2019},
+isbn = {9781450361064},
+publisher = {Association for Computing Machinery},
+address = {New York, NY, USA},
+url = {https://doi.org/10.1145/3330482.3330491},
+doi = {10.1145/3330482.3330491},
+abstract = {Due to the growth of hate speech on social media in recent years, it is important to understand this issue. An automatic hate speech detection system is needed to help to counter this problem. There have been many studies on detecting hate speech in short documents like Twitter data. But to our knowledge, research on long documents is rare, we suppose that the difficulty is increasing due to the possibility of the message of the text may be hidden. In this research, we explore in detecting hate speech on Indonesian long documents using machine learning approach. We build a new Indonesian hate speech dataset from Facebook. The experiment showed that the best performance obtained by Support Vector Machine (SVM) as its classifier algorithm using TF-IDF, char quad-gram, word unigram, and lexicon features that yield f1-score of 85%.},
+booktitle = {Proceedings of the 2019 5th International Conference on Computing and Artificial Intelligence},
+pages = {164–169},
+numpages = {6},
+keywords = {machine learning, SVM, long documents, hate speech detection},
+location = {Bali, Indonesia},
+series = {ICCAI '19}
+}
+"""
+_DATASETNAME = "id_hsd_nofaaulia"
+_LANGUAGES = ["ind"]
+_LOCAL = False
+
+_DESCRIPTION = """\
+There have been many studies on detecting hate speech in short documents like Twitter data. But to our knowledge, research on long documents is rare, we suppose that the difficulty is increasing due to the possibility of the message of the text may be hidden. In this research, we explore in detecting hate speech on Indonesian long documents using machine learning approach. We build a new Indonesian hate speech dataset from Facebook.
+"""
+
+_HOMEPAGE = "https://github.com/nofaulia/hate-speech-detection"
+
+_LICENSE = "Unknown"
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/nofaulia/hate-speech-detection/main/data/dataset.csv",
+}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+_SOURCE_VERSION = "1.0.0"
+_NUSANTARA_VERSION = "1.0.0"
+
+
+# TODO: Name the dataset class to match the script name using CamelCase instead of snake_case
+class NewDataset(datasets.GeneratorBasedBuilder):
```

Comment:
Need to update this to the dataset name and the short description as well

Reply:
Ok. Updated.